src/PatientChecking/PatientChecking/Views/Dashboard/Home.cshtml

```diff
@@ -2,4 +2,67 @@
     ViewData["Title"] = "Dashboard";
     ViewData["Page"] = "Dashboard";
 }
-<p>This is dashboard</p>
+@section css {
+    <link href="@Url.Content("~/css/Dashboard.css")" rel="stylesheet" type="text/css" />
+}
+<script>
+    (function () {
+        $('#loading-div-background').show();
```

Comment:
Can we move this loading to common file due to this file can use to other places?

Reply:
I customized Loading tag, so we can use to other places by import:  <link href="@Url.Content("~/css/Loading.css")" rel="stylesheet" type="text/css" /> to each razor view